models/Petition.model.js

```diff
@@ -23,11 +26,13 @@ const petitionSchema = Schema(
       enum: {
         values: ["receive", "provide", "deliver"],
```

Comment:
I updated to be more consistent with the type of petition/participation. The seeding broke [here](https://github.com/cqnguy23/Hackathon_Project/blob/ee0054d03df1e990bfaf119794fb35a6332dd3a0/controllers/foos.controller.js#L41) without it.

Reply:
cool